src/Moryx.Products.Model/Entities/ProductTypeEntity.cs

```diff
@@ -28,7 +28,7 @@ public class ProductTypeEntity : ModificationTrackedEntityBase
 
         public virtual ICollection<ProductTypePropertiesEntity> OldVersions { get; set; }
 
-        public virtual ProductTypePropertiesEntity CurrentVersion { get; protected internal set; }
```

Comment:
The current version is updated automatically. For which requirement do you set it manually?